cmd/deployment/create.go

```diff
@@ -167,12 +169,29 @@ var createCmd = &cobra.Command{
 			return err
 		}
 
-		return ecctl.Get().Formatter.Format("", res)
+		var track, _ = cmd.Flags().GetBool("track")
```

Comment:
I think this one deserves a common function `TrackOrReturn`. The code is almost identical and probably we will have more code like this in the future. I know we already have existing code like this so it makes more sense to keep it in a single place